app/models/content_item.rb

```diff
@@ -72,12 +70,4 @@ def whitehall_url
     return unless publishing_app == "whitehall"
     "#{WHITEHALL}/government/admin/by-content-id/#{content_id}"
   end
-
-  def add_organisations_by_id(orgs)
```

Comment:
This change doesn't seem to fit with this commit 354fad88691ba1a6abb48df4c9150fe33cd3ee96. I think the the deletion in the last commit of adding taxons needs to be swapped with the one here for adding organisations.